### ISSUE TITLE
Forcing a language

### ISSUE THREAD

**Issue by cvetelinandreev** [thomaswp/snap-replay#7]
What would be a good way to force a Language on the playback side?

I played a bit with the code and this works in some cases. However it is not a good solution:
playback.js, line 330.
`
            setTimeout(() => {
                this.snapWindow.recorder.constructor.setRecordScale(this.script.config.blockScale);
 
                setTimeout(() => {
                    this.snapWindow.recorder.constructor.setLanguage(this.script.config.language);

                    setTimeout(() => {
                        this.snapWindow.recorder.constructor.resetSnap(this.script.startXML);
                    }, 1000);    
                }, 1000);    
            }, 1000);
`

- The setTimeout avoids the issue with the async project loading in a hacky way.
- Another issue is that this code triggers reloading of the IDE 3 times. 2 times with empty project and once with the one from start.xml. I'm wondering if we can do the loading only once.

This is related somehow with #2

**Comment by cvetelinandreev** [comment]
Another observation:

resetSnap() is invoked on rewind. Since the project loading is async, the record logs are executed before the project gets loaded. The result is sporadic. Attached is a project to test with. 
1) Start the playback. 
2) Fast forward at 66%. 
3) Then rewind to 33%

[Archive.zip](https://github.com/thomaswp/snap-replay/files/7985770/Archive.zip)
